app/helpers/takeScreenshot.js

```diff
@@ -0,0 +1,27 @@
+import { BrowserWindow, remote } from 'electron'
+
+export default async function takeScreenshot (html, deviceWidth) {
+  return new Promise((resolve, reject) => {
+    const win = new remote.BrowserWindow({
+      width: deviceWidth,
+      show: false
+    })
+    
+    console.log(html)
+    
+    win.loadURL(`data:text/html, ${html}`)
+    
+    win.webContents.on('did-finish-load', () => {
+      // Window is not fully loaded after this event, hence setTimeout()...
+      win.webContents.executeJavaScript(`document.querySelector('body').getBoundingClientRect().height`, (height) => {
+        win.setSize(deviceWidth, height)
+      })
+      setTimeout(() => {
```

Comment:
Just to be safe, put the `setTimeout` in the callback of `executeJavaScript`. If app is run in, for e.g a Pentium II 700mhz (lulz), the js execution will maybe takes 501ms :smile: